app/javascript/Components/Result/result.jsx

```diff
@@ -12,6 +12,7 @@ import CreateModifyAnnotationPanel from "../Modals/create_modify_annotation_pane
 import CreateTagModal from "../Modals/create_tag_modal";
 import {pathToNode} from "../Helpers/range_selector";
 import {ResultContext} from "./result_context";
+import {annotation_context_menu} from "../../../assets/javascripts/Results/context_menu";
```

Comment:
The change is good; however, please also move the file to `app/javascript/Components/Result` (since this file is only used by the `Result` component).